src/Umbraco.Web/Install/InstallException.cs

```diff
@@ -1,44 +1,109 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using System.Runtime.Serialization;
 
 namespace Umbraco.Web.Install
 {
     /// <summary>
-    /// Used for steps to be able to return a json structure back to the UI
+    /// Used for steps to be able to return a JSON structure back to the UI.
     /// </summary>
+    /// <seealso cref="System.Exception" />
+    [Obsolete("Refactor the view model into a concrete serializable type.")]
```

Comment:
This shouldn't be obsoleted since it's still in use. I understand that it might not conform to an ideal exception type but until we refactor to something different this can't be obsoleted. We can refactor this later (baby steps... would rather get this larger PR merged in sooner than later)

Reply:
I've removed the serialization of the (possibly complex and unsupported) view model, so it at least keeps the view name (so it's better then before) 👍